.travis.yml

```diff
@@ -2,25 +2,32 @@
 #   Generic MoveIt Travis Continuous Integration Configuration File
 #   Works with all MoveIt! repositories/branches
 #   Author: Dave Coleman, Jonathan Bohren
+dist: trusty
 language:
   - cpp
   - python
 python:
   - "2.7"
 compiler:
   - gcc
+cache:
+  - apt
+  - ccache
 env:
   global:
     - ROSWS=wstool
     - BUILDER=catkin
+    - USE_DOCKER=true
+    - USE_TRAVIS=true
     - CATKIN_PARALLEL_JOBS='-p2'
     - CATKIN_PARALLEL_TEST_JOBS='-p1'
     - ROS_PARALLEL_JOBS="-j2"
     - ROS_PARALLEL_TEST_JOBS='-j1'
   matrix:
     - ROS_DISTRO=indigo  USE_DEB=true
     - ROS_DISTRO=indigo  USE_DEB=false NOT_TEST_INSTALL=true
-    - ROS_DISTRO=kinetic USE_DEB=false
+    - ROS_DISTRO=kinetic USE_DEB=false NOT_TEST_INSTALL=true
+    - ROS_DISTRO=melodic USE_DEB=false NOT_TEST_INSTALL=true BEFORE_SCRIPT="pwd; sed -i \"35iadd_definitions(-Wno-deprecated)\" hrpsys/CMakeLists.txt; (cd hrpsys; git diff)"
```

Comment:
When we remove hrpsys from `.travis.rosinstall.melodic`, we have to remove this `BEFORE_SCRIPT`

Reply:
Based on https://github.com/start-jsk/rtmros_tutorials/pull/571#discussion_r423419963, could you remove `BEFORE_SCRIPT`?

Reply:
Thank you. I fixed